### ISSUE TITLE
Update for light component (rgb) fails - error in logs

### ISSUE THREAD

**Issue by jurgen272** [iMicknl/ha-tahoma#96]
**Describe the bug**
RGB light is working correct, but I still have some errors in the loggings.

Errors:

Logboekdetails (ERROR)
Logger: homeassistant.helpers.entity
Source: custom_components/tahoma/light.py:160
First occurred: 8:06:58 (327 occurrences)
Last logged: 10:58:22

Update for light.light_terras_rgb fails
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 272, in async_update_ha_state
    await self.async_device_update()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 466, in async_device_update
    self.update  # type: ignore
  File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/config/custom_components/tahoma/light.py", line 160, in update
    CORE_BLUE_COLOR_INTENSITY_STATE
TypeError: color_RGB_to_hs() missing 2 required positional arguments: 'iG' and 'iB'

**Environment (please complete the following information):**
 - **Home Assistant version:** 0.112.0

**Device:** (if your problem is related to a specific device)
 - **Type**: io:DimmableRGBLightIOCompo

**Comment by vlebourl** [comment]
Hi, 
Thanks for the report. Can you edit file `light.py` (using File Editor add-on for instance) and change line 151 to the end with:
```
        if CORE_RED_COLOR_INTENSITY_STATE in self.tahoma_device.active_states:
            self._hs_color = color_util.color_RGB_to_hs(
                self.tahoma_device.active_states.get(
                    CORE_RED_COLOR_INTENSITY_STATE
                ),
                self.tahoma_device.active_states.get(
                    CORE_GREEN_COLOR_INTENSITY_STATE
                ),
                self.tahoma_device.active_states.get(
                    CORE_BLUE_COLOR_INTENSITY_STATE
                )
            )
```

Restart HA and let me know whether you still get the errors.
Thanks

**Comment by jurgen272** [comment]
Hi,

I changed:

        if CORE_RED_COLOR_INTENSITY_STATE in self.tahoma_device.active_states:
            self._hs_color = color_util.color_RGB_to_hs(
                [
                    self.tahoma_device.active_states.get(
                        CORE_RED_COLOR_INTENSITY_STATE
                    ),
                    self.tahoma_device.active_states.get(
                        CORE_GREEN_COLOR_INTENSITY_STATE
                    ),
                    self.tahoma_device.active_states.get(
                        CORE_BLUE_COLOR_INTENSITY_STATE
                    ),
                ]
            )

With:

       if CORE_RED_COLOR_INTENSITY_STATE in self.tahoma_device.active_states:
            self._hs_color = color_util.color_RGB_to_hs(
                self.tahoma_device.active_states.get(CORE_RED_COLOR_INTENSITY_STATE),
                self.tahoma_device.active_states.get(CORE_GREEN_COLOR_INTENSITY_STATE),
                self.tahoma_device.active_states.get(CORE_BLUE_COLOR_INTENSITY_STATE),
            )

But get en error in the loggings:

Error setting up entry xxx@xxx for light
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 220, in async_setup
    hass, self
  File "/usr/src/homeassistant/homeassistant/components/light/__init__.py", line 280, in async_setup_entry
    return await hass.data[DOMAIN].async_setup_entry(entry)
  File "/usr/src/homeassistant/homeassistant/helpers/entity_component.py", line 150, in async_setup_entry
    platform_type,
  File "/usr/src/homeassistant/homeassistant/setup.py", line 278, in async_prepare_setup_platform
    platform = integration.get_platform(domain)
  File "/usr/src/homeassistant/homeassistant/loader.py", line 322, in get_platform
    f"{self.pkg_path}.{platform_name}"
  File "/usr/local/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/config/custom_components/tahoma/light.py", line 150
    if CORE_RED_COLOR_INTENSITY_STATE in self.tahoma_device.active_states:
                                                                         ^
IndentationError: unindent does not match any outer indentation level

**Comment by vlebourl** [comment]
Python is very picky with indentation, github copy and paste might have messed it up... I think the `if` line is one space too far left. You should have 8 whitespaces before the `if`, 12 on the next line and 16 on the three lines inside parenthesis.

**Comment by jurgen272** [comment]
Indeed, indentation problem.
It is working now without errors.

**Comment by vlebourl** [comment]
Fixed in 1.5alpha2 by #97
